app/models/manageiq/providers/kubernetes/container_manager_mixin.rb

```diff
@@ -174,7 +174,7 @@ def check_policy_prevent(policy_event, event_target, cb_method)
       :class_name  => self.class.to_s,
       :instance_id => id,
       :method_name => :check_policy_prevent_callback,
-      :args        => [cb_method, event_target],
+      :args        => [cb_method, event_target.class.name, event_target.id],
```

Comment:
> I view the purpose of methods like these to dictate and protect the messages that go onto the queue.

Good point.  The ability to understand scan_job_create -> raw_scan_job_create without thinking about how check_policy_prevent() works might actually be a drawback :-|
However, if you read the following, without knowing what exactly check_policy_prevent() does:
```
   def scan_job_create(entity)
     check_policy_prevent(:request_containerimage_scan,
                          :raw_scan_job_create, entity.class.name, entity.id)
   end
```
isn't it "obvious" that a raw_scan_job_create(name, id) callback is being queued?
And if not could, could it be improved by naming (e.g. "queue_if_policy_doesnt_prevent" or something)?
(What one misses by not looking into check_policy_prevent() is that there is another level of queuing first.)

> If you are going to document what parameters are going into *cb_method then why not just document it by having actual variables?

:+1:  The function is trying to be generic, but could at least take `..., cb_method, *cb_args`.

> I view the purpose of methods like these to dictate and protect the messages that go onto the queue.

P.S. I wish there was a way to actually prevent AR serializion, instead of constrained coding style that "dictates and protects"...  I know it's a hard problem.

Anyway, feel free to override me.  My suggestion was purely cosmetic.
E.g. @simon3z's concern about migration, if it conflicts, is more important.